src/hooks/use-dhis2-deployment.ts

```diff
@@ -84,7 +84,12 @@ export const useDhis2DeploymentCreation = ({ onComplete }) => {
                 await addStackToDeployment(deploymentId, STACK_NAMES.CORE, values)
 
                 if (values[`include_${STACK_NAMES.PG_ADMIN}`]) {
-                    await addStackToDeployment(deploymentId, STACK_NAMES.PG_ADMIN, values)
+                    const { PGADMIN_CONFIRM_PASSWORD, ...valuesWithoutConfirmPassword } = values[STACK_NAMES.PG_ADMIN]
+                    const newObjectWithoutConfirmPassword = {
+                        ...values,
+                        pgadmin: valuesWithoutConfirmPassword,
```

Comment:
```suggestion
                        [STACK_NAMES.PG_ADMIN]: valuesWithoutConfirmPassword,
```

Reply:
`STACK_NAMES` could go into its own file (`src/constants.ts`), then these values could be used everywhere. Right now we have the constants here, but everywhere else we're using the strings directly. I made a comment in the other PR about constants. If we have a constants file, we could put those constants into `src/constants.ts` as well